plugins/database/src/low_level_odbc.cpp

```diff
@@ -572,7 +577,7 @@ cllExecSqlWithResult( icatSessionStruct *icss, int *stmtNum, const char *sql ) {
        needed here, and in fact causes postgres processes to be in the
        'idle in transaction' state which prevents some operations (such as
        backup).  So this was removed. */
-    rodsLog( LOG_DEBUG10, "%s", sql );
+    log_sql::debug("{}: {}", __func__, sql);
```

Comment:
Add more context to the messages so the reader understands what is being conveyed. For example: 
```
{}: SQL = [{}]
```